portfolio/src/main/webapp/script.js

```diff
@@ -48,6 +51,18 @@ function OnWindowScrolled() {
     }
 }
 
+/**
+ * Callback when comments-container is scrolled
+ */
+function OnCommentsContainerScrolled() {
+    // Get the comments container element
+    const commentsElement = document.getElementById('comments-container');
+
+    if(commentsElement.scrollTop === 0) {
```

Comment:
I'm a bit confused by this. Could you add a comment about what that condition means from a user perspecitve?